### PR TITLE
Fix rate limit err, split origin and add stuff

### PR DESCRIPTION
**Pull request by grongor**
- API client now limits itself to not hit rate limits
 - Origin resource was split into separate resources by type (MoveResource was implemented to smoothen the transition)
 - Origin and cluster URLs now have url and url_part attributes, so that clients can access which ever one is more convenient
 - Cdn and Origin "all" resource now have all available attributes (no longer limited by the "list" endpoint)

### CODE REVIEW

File: docs/data-sources/cdn.md [hunk 1]
```diff
@@ -137,7 +136,7 @@ Read-Only:
 Read-Only:
 
 - `ssl_id` (String) ID (UUID) of the SSL certificate
-- `type` (String) Available values: instantSsl, none, SNI
+- `type` (String) Possible values: instantSsl, none, SNI, SAN
```
Comment (fialamarek) [inline docs/data-sources/cdn.md:139]: ten SAN imho uz vubec nenabizime

File: docs/data-sources/cdns.md [hunk 2]
```diff
@@ -29,11 +29,127 @@ data "cdn77_cdns" "all" {
...
+</ul>
+
+
+<a id="nestedatt--cdns--ssl"></a>
+### Nested Schema for `cdns.ssl`
+
+Read-Only:
+
+- `ssl_id` (String) ID (UUID) of the SSL certificate
+- `type` (String) Possible values: instantSsl, none, SNI, SAN
```
Comment (fialamarek) [inline docs/data-sources/cdns.md:142]: SAN bych vyhodil

File: docs/resources/cdn.md [hunk 3]
```diff
@@ -142,7 +141,7 @@ Optional:
 Optional:
 
 - `ssl_id` (String) ID (UUID) of the SSL certificate
-- `type` (String) Available values: instantSsl, none, SNI
+- `type` (String) Possible values: instantSsl, none, SNI, SAN
```
Comment (fialamarek) [inline docs/resources/cdn.md:144]: Pryc SAN

**Review by fialamarek — COMMENTED** [review]
doc is OK

File: docs/data-sources/cdn.md (hunk 1 above)
Comment (grongor) [reply docs/data-sources/cdn.md:139]: Why we have it in the api then? https://client.cdn77.com/support/api-reference/v3/cdn-resources#edit-cdn-resource

**Review by tomachine — APPROVED** [review]

**Pull request merged**